### PR TITLE
Add downloads count to CSV

### PR DESCRIPTION
**Pull request by ThibaudDauce**
See https://github.com/etalab/data.gouv.fr/issues/1237

### CODE REVIEW

File: udata/core/dataset/models.py [hunk 1]
```diff
@@ -718,6 +718,11 @@ def quality(self):
 
         result['score'] = self.compute_quality_score(result)
         return result
+    
+    @property
+    def downloads(self):
+        return sum(map(lambda o: o.metrics.get('views', 0), self.resources))
```
Comment (maudetes) [inline udata/core/dataset/models.py:724]: We mostly use list comprehension in udata code
```suggestion
        return sum(resource.metrics.get('views', 0) for resource in self.resources)
```

File: udata/core/dataset/csv.py [hunk 2]
```diff
@@ -33,6 +33,7 @@ class DatasetCsvAdapter(csv.Adapter):
         ('tags', lambda o: ','.join(o.tags)),
         ('archived', lambda o: o.archived or False),
         ('resources_count', lambda o: len(o.resources)),
+        'downloads',
```
Comment (maudetes) [inline udata/core/dataset/csv.py:36]: I wonder if naming it `resources_downloads` would make it more explicit, following `resources_count`?

**Review by maudetes — APPROVED** [review]
I have only cosmetic comments :)

Don't forget the changelog entry ;)

File: udata/core/dataset/csv.py (hunk 2 above)
Comment (ThibaudDauce) [reply udata/core/dataset/csv.py:36]: I use the term requested in the issue https://github.com/etalab/data.gouv.fr/issues/1237 but I can change it.

Comment (maudetes) [reply udata/core/dataset/csv.py:36]: Let's keep with `downloads` :+1:

**Review by maudetes — APPROVED** [review]
Don't forget the changelog ;)

**Pull request merged**